### PR TITLE
All fixes for TB60 from standard release

### PR DESCRIPTION
**Pull request by jobisoft**
These are all my commits for the standard version applied to the pro version.  

I will try to fix the [[CURSOR]] bug for both versions. After that, I will only work on the pro version and after it runs fine, I will replace the standard version with it.

### CODE REVIEW

File: chrome/content/quicktext.js [hunk 1]
```diff
@@ -109,21 +109,25 @@ var quicktext = {
...
-    if (document.getElementById("date-long"))
-      document.getElementById("date-long").setAttribute("label", this.mStringBundle.getFormattedString("date", [dateTimeService.FormatDate("", dateTimeService.dateFormatLong, timeStamp.getFullYear(), timeStamp.getMonth()+1, timeStamp.getDate())]));
-    if (document.getElementById("time-noseconds"))
-      document.getElementById("time-noseconds").setAttribute("label", this.mStringBundle.getFormattedString("time", [dateTimeService.FormatTime("", dateTimeService.timeFormatNoSeconds, timeStamp.getHours(), timeStamp.getMinutes(), timeStamp.getSeconds())]));
-    if (document.getElementById("time-seconds"))
-      document.getElementById("time-seconds").setAttribute("label", this.mStringBundle.getFormattedString("time", [dateTimeService.FormatTime("", dateTimeService.timeFormatSeconds, timeStamp.getHours(), timeStamp.getMinutes(), timeStamp.getSeconds())]));
+    var options = {};
+	options["date-short"] = { year: "numeric", month: "2-digit", day: "2-digit" };
```
Comment (lieser) [inline chrome/content/quicktext.js]: use spaces here instead of tabs

File: components/wzQuicktextVar.js [hunk 2]
```diff
@@ -1100,10 +1106,10 @@ if (XPCOMUtils.generateNSGetFactory)
 else
   var NSGetModule = XPCOMUtils.generateNSGetModule([wzQuicktextVar]);
 
-if (!kDebug)
-  debug = function(m) {};
-else
-  debug = function(m) {dump("\t *** wzQuicktextVar: " + m + "\n");};
+var debug = kDebug ?  function(m) {dump("\t *** wzQuicktext: " + m + "\n");} : function(m) {};
+
```
Comment (lieser) [inline components/wzQuicktextVar.js]: I would only add max 1 new newline (like in the master branch), not 3

File: install.rdf [hunk 3]
```diff
@@ -5,10 +5,11 @@
   <Description about="urn:mozilla:install-manifest">
     <em:id>quicktextpro@extensions.hesslow.se</em:id>
     <em:name>Quicktext Pro</em:name>
-    <em:version>0.9.11.7</em:version>
+    <em:version>0.9.12</em:version>
```
Comment (lieser) [inline install.rdf]: Is it intended the the normal version (in master) has  `0.9.11.12`, and the pro will be at `0.9.12`?

Comment (jobisoft) [reply install.rdf]: Master is also at 0.9.12

Comment (lieser) [reply install.rdf]: You are right, I looked at the wrong place.

**Comment by jobisoft** [comment]
Is it ok now?

**Comment by lieser** [comment]
Yes, looks good now. And again, thanks a lot for fixing this add-on.

**Pull request merged**